Added simple action descriptions

uwlaten opened the pull request.

uwlaten committed:
Altered comment in ActionController.java

uwlaten committed:
Altered comment in ActionController.java

uwlaten committed:
Minor corrections to spelling.
Added simple action report system: each action has a description which is reported in the UI upon completion of the action.

bjedrzejewski commented:
@andrewbissette I can see that you have some merge conflicts that you must resolve first.

bjedrzejewski commented on src/main/java/com/bjedrzejewski/action/ScreamAction.java:
```diff
@@ -0,0 +1,49 @@
+import java.util.Map;
+
+/**
+ * Created by bartoszjedrzejewski on 06/08/2016.
```
The comment on this class is wrong.

bjedrzejewski commented on src/main/java/com/bjedrzejewski/game/DayTime.java:
```diff
@@ -21,7 +21,7 @@
         @Override
         public String getDescription() {
-            return "It is Morning.";
+            return "It is Morning. ";
```
Why did you add extra space? I think this could be handled better in the html.

bjedrzejewski commented on src/main/java/com/bjedrzejewski/game/GameState.java:
```diff
@@ -86,12 +89,25 @@ public void advanceTime(){
         String description = "";
 
+        // add report of last action
+        description += getLastActionDescription();
```
Adding spaces or new lines could be better done here.

bjedrzejewski commented on src/main/java/com/bjedrzejewski/location/beach/ExploreTheBeach.java:
```diff
@@ -12,6 +12,7 @@
 public class ExploreTheBeach implements PlayerAction{
 
     private static final Logger log = Logger.getLogger(ExploreTheBeach.class);
+    private static final String actionDescription = "You explore the beach for a while. ";
```
Let's avoid ending with spaces.

bjedrzejewski commented:
Hey @andrewbissette overall- very good pull request. Just need to fix the merge issue and address a few very minor comments I added.

uwlaten committed:
Corrected comment in ScreamAction.java
Removed spaces from descriptions (with one exception, see comment on GameController.java)

uwlaten committed:
Merge branch 'master' into andrew-test

uwlaten commented:
All should be resolved now.

bjedrzejewski commented:
Thanks for the pr 👍
